src/app/fdctl/monitor/helper.c

```diff
@@ -174,3 +175,29 @@ printf_pct( char ** buf,
   if( pct<=999.999 ) { PRINT( " %7.3f", pct ); return; }
   /**/                 PRINT( ">999.999" );
 }
+
+
+int
+fd_getch()
+{
+  struct termios oldt, newt;
+  int ch;
+  int oldf;
+ /* Disables character echo and canonical mode since we want the input to be processes immediately.
+  * Terminal also set to non blocking in case the user doesn't send any input.
+  * */
+  tcgetattr(STDIN_FILENO, &oldt);
+  newt = oldt;
+  newt.c_lflag &= (tcflag_t)~(ICANON | ECHO);
+  tcsetattr(STDIN_FILENO, TCSANOW, &newt);
```

Comment:
> I wonder if we can leave this enabled all the time? 🤔 I'm not sure whether that's a good idea though since it requires extra care to restore the terminal when the user exits.

The terminal restores back fine even if I leave it on so I don't see any issues if you think that's better. It would save us calls to ioctl.